Kickstarter-iOS/Views/Controllers/SheetOverlayContainerViewController.swift

```diff
@@ -0,0 +1,59 @@
+import UIKit
+import Library
+import Prelude
+
+/**
+ SheetOverlayContainerViewController intended to be used as a container for another view controller
+ that you would like to render as a "sheet" or "card" that partially covers the content beneath it.
+ */
+
+final class SheetOverlayContainerViewController: UIViewController {
+  private let transitionAnimator = SheetOverlayTransitionAnimator()
+
+  init(childViewController: UIViewController, childViewOffset: CGFloat) {
+    super.init(nibName: nil, bundle: nil)
+
+    _ = self
+      |> \.modalPresentationStyle .~ .custom
+      |> \.transitioningDelegate .~ self
+
+    self.addChild(childViewController)
```

Comment:
Interestingly, moving the child VC setup to the `viewDidLoad` actually breaks the animation 🤔The Apple docs don't specify when the right time to configure a child-parent relationship programmatically is, but they do say this about about configuration using interface builder: `"The children must be instantiated at the same time as the parent so that the appropriate parent-child relationships can be created."` Is it possible that this is also true for programmatic setup?

Reply:
^ scratch that, the `modalPresentationStyle` is what needs to be set in the `init`.